test/example.css

```diff
@@ -2,6 +2,10 @@ html {
     box-sizing: border-box;
 }
 
+a {
+    animation: 100ms;
```

Comment:
@giladgray Just noticed that `animation` is not part of `property-order.js` (haven't touched this file) so I'm not sure how you want me to do this change. Should `animation` be added to the property order file? If so, where?